test/karma.conf.js

```diff
@@ -25,18 +23,22 @@ module.exports = (config) => {
       {pattern: 'node_modules/zone.js/dist/jasmine-patch.min.js', included: true, watched: false},
       {pattern: 'node_modules/zone.js/dist/async-test.js', included: true, watched: false},
       {pattern: 'node_modules/zone.js/dist/fake-async-test.js', included: true, watched: false},
+      {pattern: 'node_modules/hammerjs/hammer.min.js', included: true, watched: false},
```

Comment:
Same as for Material, do you use hammer as well as moment in unit tests?

Reply:
Most likely not, will remove